test/cql-pytest/test_keyspace.py

```diff
@@ -97,15 +97,22 @@ def test_drop_keyspace_nonexistent(cql):
         cql.execute('DROP KEYSPACE nonexistent_keyspace')
 
 # Test trying to ALTER a keyspace.
-def test_alter_keyspace(cql, this_dc):
+def test_alter_keyspace(cql, this_dc, has_tablets):
     with new_test_keyspace(cql, "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }") as keyspace:
-        cql.execute(f"ALTER KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 3 }} AND DURABLE_WRITES = false")
+        if has_tablets:
+            repl_opts = ""
+        else:
+            repl_opts = f" REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 3 }} AND "
```

Comment:
nit: to prepare for future support for rf changes with tablets we can change rf here only by +1 to 2 rather than 3.

Reply:
I think it should be changed in the PR that introduces this behaviour. Otherwise I'd receive a comment that I'm introducing a not relevant change. But thanks for pointing that out.